lib/rules/ssr/ssr-no-form-factor.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
         docs: {
             description: 'Discourage use of formFactor.',
             category: 'LWC',
-            url: docUrl('ssr/no-form-factor'),
+            url: docUrl('docs/rules/ssr/ssr-no-form-factor'),
```

Comment:
`docs/rules/` is prepended by [docUrl](https://github.com/salesforce/eslint-plugin-lwc/blob/master/lib/util/doc-url.js#L14) routine. Same comment for all the other docUrls changes below.
```suggestion
            url: docUrl('ssr/ssr-no-form-factor'),
```

Reply:
Fixed it myself and requested @nolanlawson to review and approve.